packages/bundle-source/src/script.js

```diff
@@ -89,13 +92,33 @@ export async function bundleScript(
     workspaceModuleLanguageForExtension,
     moduleTransforms,
     sourceMapHook,
+    useEvaluate: moduleFormat === 'nestedEvaluate',
+    sourceUrlPrefix: '/bundled-source/.../',
+    format:
+      moduleFormat === 'nestedEvaluate' || moduleFormat === 'getExport'
+        ? 'cjs'
+        : undefined,
   });
 
+  if (moduleFormat === 'endoScript') {
+    source = `${source}()`;
```

Comment:
For parity and resiliency:
```suggestion
    source = `(${source})()`;
```